### PR TITLE
make test work on 32-bit Debian

### PR DESCRIPTION
**Pull request by kroening**

### CODE REVIEW

**Review by tautschnig — APPROVED** [review]
That shouldn't be "Debian" specific, though? Just any 32-bit system, even Windows?

File: regression/cbmc/memcpy1/test.desc [hunk 1]
```diff
@@ -4,7 +4,7 @@ main.c
 ^EXIT=10$
 ^SIGNAL=0$
 ^VERIFICATION FAILED$
-\[(__builtin___memcpy_chk|memcpy)\.pointer_dereference\.16\] dereference failure: pointer outside object bounds in \*\(\(\(const char \*\)src \+ \(signed long (long )?int\)n\) - \(signed long (long )?int\)1\): FAILURE$
+\[(__builtin___memcpy_chk|memcpy)\.pointer_dereference\.16\] dereference failure: pointer outside object bounds in \*\(\(\(const char \*\)src \+ \(signed (long (long )?)?int\)n\) - (\(signed (long )?int\))?1\): FAILURE$
```
Comment (tautschnig) [inline regression/cbmc/memcpy1/test.desc]: AppVeyor serves as reminder that a `long` got lost in this change.

Comment (kroening) [reply regression/cbmc/memcpy1/test.desc]: Fixed

**Comment by kroening** [comment]
I think this is any 32-bit system with the exception of Windows.

**Pull request merged**